[2015-03-13 00:02:30,218][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 00:06:30,815][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 00:10:31,419][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 00:14:32,020][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 00:18:32,618][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 00:22:33,217][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 00:22:43,228][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x2ca84737]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:788)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 00:22:43,230][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x9b541252]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:788)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,232][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xb222c191]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:788)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,248][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xe784567a]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:788)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,254][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xa963642b]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:791)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,260][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x5275bfba]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:791)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,269][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xe4ab5483]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:791)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,271][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x2ca05f72]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,273][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xeb987d4f]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,281][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x89085bf5]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,284][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x2227e88d]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,286][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x2e679f50]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,288][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xb29adf9c]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,290][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xaedc149a]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:797)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:48:18,292][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x9dbddcbc]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:800)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,434][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x31ec059b]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:788)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,435][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x51ee1b03]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:788)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,436][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x7d6fba3c]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:791)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,437][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x29fc9ad8]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:791)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,438][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x24681fa7]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:791)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,444][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x1f81b400]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,447][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xf47324ee]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,448][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x2cad5ad4]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,449][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x34e3c9cf]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,451][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x3c7bb079]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,453][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xc4730c7b]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,455][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x45fca768]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:797)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,456][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x5d47377d]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:800)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:49:48,457][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:788)
	... 7 more
[2015-03-13 08:50:38,528][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x91e97d51]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:788)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,530][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x52540471]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:788)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,531][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xa6c3be94]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:791)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,532][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xf6d37fd0]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:791)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,533][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x87d46189]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:791)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,534][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x4ae37317]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,535][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x733374db]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,536][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xffb6121c]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,537][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x3e8f1181]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,537][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xaeb1ab80]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,538][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x8fdc1019]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:794)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,540][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0xe80eff2d]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:797)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:50:38,541][WARN ][transport.netty          ] [Caliban] exception caught on transport layer [[id: 0x56a1e07f]], closing connection
java.net.SocketException: Host is down
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:465)
	at sun.nio.ch.Net.connect(Net.java:457)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:670)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:800)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
[2015-03-13 08:52:33,571][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: java.net.ConnectException: Connection refused: /10.168.96.119:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:739)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:152)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 08:53:48,654][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: java.net.ConnectException: Connection refused: /10.168.96.119:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:739)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:152)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 08:55:01,970][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: java.net.ConnectException: Connection refused: /10.168.96.119:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:739)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:152)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 08:57:35,491][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: java.net.ConnectException: Connection refused: /10.168.96.119:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:739)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:152)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 08:59:37,315][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:03:37,919][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:07:38,518][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:11:39,116][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:15:39,716][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:19:40,319][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:23:40,918][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:27:41,518][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:31:42,119][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:35:42,719][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:39:43,317][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:43:43,917][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:47:44,519][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:51:45,121][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:55:45,716][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 09:59:46,316][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:03:46,919][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:07:47,520][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:11:48,118][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:15:23,009][INFO ][cluster.metadata         ] [Caliban] [movies] creating index, cause [auto(index api)], shards [5]/[1], mappings [movie]
[2015-03-13 10:15:23,135][INFO ][cluster.metadata         ] [Caliban] [movies] update_mapping [movie] (dynamic)
[2015-03-13 10:15:48,717][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:19:49,316][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:23:49,918][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:27:50,516][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:31:51,121][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:35:51,717][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:39:52,318][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:43:52,917][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:47:55,006][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:52:46,718][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 10:56:47,320][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 11:00:47,919][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 11:04:48,516][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 11:08:49,119][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 11:12:49,719][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 11:16:50,316][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 11:20:50,917][WARN ][cluster.service          ] [Caliban] failed to reconnect to node [Caliban][Og0pE1lMRH2q9ANSfdKiDw][mjrana.local][inet[/10.168.96.119:9300]]
org.elasticsearch.transport.ConnectTransportException: [Caliban][inet[/10.168.96.119:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:807)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:741)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:714)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:150)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:544)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: org.elasticsearch.common.netty.channel.ConnectTimeoutException: connection timed out: /10.168.96.119:9300
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2015-03-13 11:24:11,950][INFO ][node                     ] [Cain Marko] version[1.4.4], pid[59180], build[c88f77f/2015-02-19T13:05:36Z]
[2015-03-13 11:24:11,950][INFO ][node                     ] [Cain Marko] initializing ...
[2015-03-13 11:24:11,954][INFO ][plugins                  ] [Cain Marko] loaded [], sites []
[2015-03-13 11:24:14,772][INFO ][node                     ] [Cain Marko] initialized
[2015-03-13 11:24:14,772][INFO ][node                     ] [Cain Marko] starting ...
[2015-03-13 11:24:14,846][INFO ][transport                ] [Cain Marko] bound_address {inet[/0:0:0:0:0:0:0:0:9302]}, publish_address {inet[/10.168.96.113:9302]}
[2015-03-13 11:24:14,862][INFO ][discovery                ] [Cain Marko] elasticsearch/helnwKxVSryrbJ96upsvzw
[2015-03-13 11:24:44,867][WARN ][discovery                ] [Cain Marko] waited for 30s and no initial state was set by the discovery
[2015-03-13 11:24:44,876][INFO ][http                     ] [Cain Marko] bound_address {inet[/0:0:0:0:0:0:0:0:9201]}, publish_address {inet[/10.168.96.113:9201]}
[2015-03-13 11:24:44,876][INFO ][node                     ] [Cain Marko] started
[2015-03-13 11:24:47,896][INFO ][cluster.service          ] [Cain Marko] new_master [Cain Marko][helnwKxVSryrbJ96upsvzw][mjrana.local][inet[/10.168.96.113:9302]], reason: zen-disco-join (elected_as_master)
[2015-03-13 11:24:47,926][INFO ][gateway                  ] [Cain Marko] recovered [0] indices into cluster_state
[2015-03-13 11:25:14,102][INFO ][node                     ] [Cain Marko] stopping ...
[2015-03-13 11:25:14,108][INFO ][node                     ] [Caliban] stopping ...
[2015-03-13 11:25:14,143][INFO ][node                     ] [Caliban] stopped
[2015-03-13 11:25:14,143][INFO ][node                     ] [Caliban] closing ...
[2015-03-13 11:25:14,154][INFO ][node                     ] [Caliban] closed
[2015-03-13 11:25:21,494][INFO ][node                     ] [Cain Marko] stopped
[2015-03-13 11:25:21,494][INFO ][node                     ] [Cain Marko] closing ...
[2015-03-13 11:25:21,498][INFO ][node                     ] [Cain Marko] closed
[2015-03-13 11:25:48,164][INFO ][node                     ] [Krang] version[1.4.4], pid[59204], build[c88f77f/2015-02-19T13:05:36Z]
[2015-03-13 11:25:48,164][INFO ][node                     ] [Krang] initializing ...
[2015-03-13 11:25:48,168][INFO ][plugins                  ] [Krang] loaded [], sites []
[2015-03-13 11:25:50,205][INFO ][node                     ] [Krang] initialized
[2015-03-13 11:25:50,205][INFO ][node                     ] [Krang] starting ...
[2015-03-13 11:25:50,254][INFO ][transport                ] [Krang] bound_address {inet[/0:0:0:0:0:0:0:0:9300]}, publish_address {inet[/10.168.96.113:9300]}
[2015-03-13 11:25:50,267][INFO ][discovery                ] [Krang] elasticsearch/oBw71olLR0iOblTl9VAryQ
[2015-03-13 11:25:54,054][INFO ][cluster.service          ] [Krang] new_master [Krang][oBw71olLR0iOblTl9VAryQ][mjrana.local][inet[/10.168.96.113:9300]], reason: zen-disco-join (elected_as_master)
[2015-03-13 11:25:54,073][INFO ][http                     ] [Krang] bound_address {inet[/0:0:0:0:0:0:0:0:9200]}, publish_address {inet[/10.168.96.113:9200]}
[2015-03-13 11:25:54,073][INFO ][node                     ] [Krang] started
[2015-03-13 11:25:54,607][INFO ][gateway                  ] [Krang] recovered [4] indices into cluster_state
[2015-03-13 11:28:16,967][INFO ][cluster.service          ] [Krang] added {[logstash-mjrana.local-59230-2010][c_TSkvTmTyey6VramoisvQ][mjrana.local][inet[/10.168.96.113:9301]]{client=true, data=false},}, reason: zen-disco-receive(join from node[[logstash-mjrana.local-59230-2010][c_TSkvTmTyey6VramoisvQ][mjrana.local][inet[/10.168.96.113:9301]]{client=true, data=false}])
[2015-03-13 11:28:21,311][INFO ][cluster.metadata         ] [Krang] [logstash-2015.03.13] creating index, cause [auto(bulk api)], shards [5]/[1], mappings [_default_]
[2015-03-13 11:28:21,447][INFO ][cluster.metadata         ] [Krang] [logstash-2015.03.13] update_mapping [logs] (dynamic)
[2015-03-13 15:22:35,532][INFO ][node                     ] [Krang] stopping ...
[2015-03-13 15:22:35,623][INFO ][node                     ] [Krang] stopped
[2015-03-13 15:22:35,623][INFO ][node                     ] [Krang] closing ...
[2015-03-13 15:22:35,628][INFO ][node                     ] [Krang] closed
[2015-03-13 15:30:38,557][INFO ][node                     ] [Fight-Man] version[1.4.4], pid[60025], build[c88f77f/2015-02-19T13:05:36Z]
[2015-03-13 15:30:38,557][INFO ][node                     ] [Fight-Man] initializing ...
[2015-03-13 15:30:38,563][INFO ][plugins                  ] [Fight-Man] loaded [], sites []
[2015-03-13 15:30:41,481][INFO ][node                     ] [Fight-Man] initialized
[2015-03-13 15:30:41,481][INFO ][node                     ] [Fight-Man] starting ...
[2015-03-13 15:30:41,552][INFO ][transport                ] [Fight-Man] bound_address {inet[/0:0:0:0:0:0:0:0:9300]}, publish_address {inet[/10.168.96.113:9300]}
[2015-03-13 15:30:41,568][INFO ][discovery                ] [Fight-Man] elasticsearch/NmRXnzBYSG6z38BUSGGITA
[2015-03-13 15:30:45,356][INFO ][cluster.service          ] [Fight-Man] new_master [Fight-Man][NmRXnzBYSG6z38BUSGGITA][mjrana.local][inet[/10.168.96.113:9300]], reason: zen-disco-join (elected_as_master)
[2015-03-13 15:30:45,378][INFO ][http                     ] [Fight-Man] bound_address {inet[/0:0:0:0:0:0:0:0:9200]}, publish_address {inet[/10.168.96.113:9200]}
[2015-03-13 15:30:45,378][INFO ][node                     ] [Fight-Man] started
[2015-03-13 15:30:45,968][INFO ][gateway                  ] [Fight-Man] recovered [5] indices into cluster_state
[2015-03-13 15:31:42,538][INFO ][cluster.service          ] [Fight-Man] added {[logstash-mjrana.local-60042-2010][wHr34op_QcOuHXhz2iJ2Hw][mjrana.local][inet[/10.168.96.113:9301]]{client=true, data=false},}, reason: zen-disco-receive(join from node[[logstash-mjrana.local-60042-2010][wHr34op_QcOuHXhz2iJ2Hw][mjrana.local][inet[/10.168.96.113:9301]]{client=true, data=false}])
